src/Hero/Hero.component.tsx

```diff
@@ -0,0 +1,176 @@
+// REACT
+import * as React from 'react';
+
+// VENDOR
+import styled from 'styled-components';
+import classnames from 'classnames';
+// ANCHOR
+import {
+    Typography,
+    TypographyProps,
+} from '../Typography/Typography.component';
+import { Grid } from '..';
+import { colors } from '../theme';
+
+// Hero
+// ------------------------------------------------------------------------------------------------------------------
+
+interface StyledHeroProps {
+    background?: string;
+    color?: string;
+    align?: string;
+    minHeight?: string;
+}
+
+const StyledHero = styled.section<StyledHeroProps>`
+    background: ${props => props.background};
+    position: relative;
+    width: 100%;
+    margin: 0 auto;
+    display: block;
+    color: ${props => props.color};
+    text-align: ${props => props.align};
+    min-height: ${props => props.minHeight};
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+`;
+
+interface HeroProps {
+    background?: string;
+    align?: string;
+    color?: string;
+    className?: string;
+    children?: any;
+    minHeight?: string;
+    height?: string;
+}
+
+export const Hero = ({
+    background = 'transparent',
+    align = 'center',
+    color = colors.white.base,
+    minHeight = '7.5rem',
+    className,
+    children,
+    ...props
+}: HeroProps): React.ReactElement<any> => (
+    <StyledHero
+        className={classnames('anchor-hero', className)}
+        background={background}
+        align={align}
+        color={color}
+        minHeight={minHeight}
+        {...props}
+    >
+        {children}
+    </StyledHero>
+);
+
+export default Hero;
+
+// Hero.Title
+// ------------------------------------------------------------------------------------------------------------------
+
+const StyledTitle = styled(Typography)`
+    text-shadow: 0 2px 4px rgba(0, 0, 0, 0.09);
+    position: relative;
+    // https://github.com/styled-components/styled-components/issues/1816
+    && {
+        // margin: 0.375rem 0;
+    }
+`;
+
+export const HeroTitle = ({
+    className,
+    children,
+    tag = 'h1',
+    weight = 600,
+    color = 'inherit',
+    align = 'inherit',
+    size = 2,
+    ...props
+}: TypographyProps): React.ReactElement<any> => (
+    <StyledTitle
+        className={classnames('anchor-hero-title', className)}
+        tag={tag}
+        color={color}
+        weight={weight}
+        align={align}
+        size={size}
+        {...props}
+    >
+        {children}
+    </StyledTitle>
+);
+
+Hero.Title = HeroTitle;
```

Comment:
This sub components are importable both like:
```js
import { HeroTitle } from './Hero';
```
and:
```js
import { Hero } from './Hero';
const { Title } = Hero;
```
Anyone have opinions on if we should enforce just one way or the other?

Reply:
Both is cool for these types of exports, but I have wholly deprecated default exports and will not allow them back. They break our bundler's chunking engine.